### ISSUE TITLE
ReadmeOSS / product documentation: use alphabetical sorting

### ISSUE THREAD

**Issue by mcjaeger** [sw360/sw360portal#294]
For larger projects, the sorting is not so easy because the licenses and involved releases are not easy to capture. Proposed improvements for the HTML (and other text and docx) generation: 

* sort releases by alphabet
* write license short ID (SPDX ID) plus number instead of number only
* sort licenses by alphabet

**Comment by AndreaVogler** [comment]
•sort releases by alphabet
--> Please sort by component name only, not by a combination of vendor / component name

**Comment by mcjaeger** [comment]
After some review of the data on test systems we have some consistency problem, because some are entering vendor name or something similar at the name of the component ("Java Security", "Apache Commons"), it appears to be a bigger confusion to sort by component name only.

Current review:

1. Project: Clearing Status: Component name (can be chosen)
2. Linked Release Hierarchy: Component name (fixed, within tree level, vendor is separate column)
3. Linked Releases and projects: Vendor + Component name + Version (fixed, within tree level)
4. Generate License Info:  Vendor + Component name (fixed, within tree level, same as no. 3)
5. Genrated Text: Not sorted for humans
6. Genrated XHTML: Not sorted for humans

-> 

1. Project: Clearing Status: One column with string of "Vendor + Component name + version", rename column to "Name"
2. Linked Release Hierarchy -> Vendor + Component + Version (make 3 columns to one)
3. Linked Releases and projects: Vendor + Component + Version (make 2 columns to one)
4. Generate License Info: no change
5. Genrated Text: sort by "full name" of Vendor + Component name + Version
6. Genrated XHTML: sort by "full name" of Vendor + Component name + Version

In a nutshell: the full name or a release, and thus sortable item for all is

Vendor + Component name + version -> generate one string, that is sorted.

**Comment by alex-evo** [comment]
@mcjaeger, I think displaying and sorting of releases it's better to use vendor's shortname rather than fullname. Do you agree?

**Comment by AndreaVogler** [comment]
Can we please have a telco to discuss issues ?
I currently do not understand your issues. For me we just need to sort by release name and nothing more.

**Comment by mcjaeger** [comment]
@alex-evo yes I agree

**Comment by mcjaeger** [comment]
closed after tickets discussion, because in fact the database seems to be always in transition mode, meaning that there is a considerable percentage without vendor information
